signal_processing/compare_signals.py

```diff
@@ -0,0 +1,66 @@
+import pickle
+import pandas as pd
+from custom_processing import process_hardware_ecg, apply_wavelet_reconstruction_denoising, apply_butter_low_pass,\
+    remove_baseline_wander, remove_mean
+from feature_extraction import compute_snr, compute_mean_ptp, compute_qrs_ratio, compute_power, compute_max_psd_freq
+
+# Load PTB data
+with open('raw_ptb_data', 'rb') as f:
+    raw_ptb_data = pickle.load(f)
+    f.close()
+
+# Create df to store analyzed signals
+healthy_controls = ['patient155', 'patient263', 'patient264', 'patient170', 'patient182']
+our_data = ['simona', 'cesar']
+df = pd.DataFrame(columns=healthy_controls+our_data)
+
+# Extract data for 5 healthy controls: patient155, patient263, patient264, patient170, patient182 and save to df
+for hc in healthy_controls:
+    patient = raw_ptb_data[hc]
+    recording = list(patient.keys())[0]
+    signal = patient[recording]['i']
+    df[hc] = signal
+
+# Add our own raw data to df: downsample and take middle 115.2 secs to match length of PTB data
+lower_bound = 60 * 1000
+upper_bound = int((60+115.2) * 1000)
+simona_sitting =\
+    process_hardware_ecg(pd.read_csv('../ecg_acquisition/raw_ecgs/simona_sitting.csv'))[lower_bound:upper_bound]
+df['simona'] = simona_sitting
+cesar_sitting =\
+    process_hardware_ecg(pd.read_csv('../ecg_acquisition/raw_ecgs/cesar_sitting.csv'))[lower_bound:upper_bound]
+df['cesar'] = cesar_sitting
+
+# Perform signal processing on all signals in df
+# Apply denoising filters on signal: use lower cutoff freq for our raw data because it's noisier
```

Comment:
Might not be bad to wrap this up into a function later. something like pre_process_raw_ecg which is what we will use when we are classifying the signals in real time